packages/webpack-pwa-manifest-plugin/tests/runTest.js

```diff
@@ -0,0 +1,67 @@
+const webpack = require('webpack');
+const path = require('path');
+const ls = require('list-directory-contents');
+const fs = require('fs');
+const assert = require('assert');
+const chalk = require('chalk');
+
+function run(name, next) {
+  console.time(name);
+  if (!name) {
+    return console.log('End.');
```

Comment:
the intent is not to return whatever console.log returns (in practice, undefined)
```suggestion
   console.log('End.');
   return;
```